packages/api/cms-api/src/page-tree/blocks/dam-file-download-link.block.ts

```diff
@@ -0,0 +1,109 @@
+import {
+    AnnotationBlockMeta,
+    BlockContext,
+    BlockData,
+    BlockField,
+    BlockInput,
+    BlockMetaField,
+    BlockMetaFieldKind,
+    createBlock,
+    inputToData,
+    TraversableTransformResponse,
+} from "@comet/blocks-api";
+import { IsEnum, IsOptional, IsUUID } from "class-validator";
+import { FilesService } from "src/dam/files/files.service";
+
+export enum OpenFileTypeMethod {
```

Comment:
Is this something that's required for every project, or for only one specific project. @thomasdax98 what do you think?

Reply:
I'm not really sure when the "New Tab" option is needed. @F10Ragger can you elaborate on the use case for the "New Tab" option?

Reply:
I believe that if the document is longer or if the primary aim is to present a PDF file nicely for the user, this option is often used.

Reply:
I see, I think that actually makes sense to have in every project